module/enrichers.mjs

```diff
@@ -49,6 +49,7 @@ async function enrichString(match, options) {
     case "skill":
     case "tool": return enrichCheck(config, label, options);
     case "save": return enrichSave(config, label, options);
+    case "item": return enrichItem(config, label);
     case "embed": return enrichEmbed(config, label, options);
```

Comment:
Just trying to keep some semblance of alphabetical order, though obviously that is tough given the way the switch fall-throughs are set up here :).

```suggestion
    case "embed": return enrichEmbed(config, label, options);
    case "item": return enrichItem(config, label);
```